tasks.py

```diff
@@ -268,6 +262,56 @@ def build(_c, debug=False):
         image.tag(constants.IMAGE, tag=latest_tag)
 
 
+@task
+def sbom(_c, debug=False):
+    """Generate an SBOM"""
+    if debug:
+        getLogger().setLevel("DEBUG")
+
+    for variant in constants.VARIANTS:
+        versioned_tag = CONTEXT[variant]["buildargs"]["VERSION"]
+        image_and_tag = f"{constants.IMAGE}:{versioned_tag}"
+
+        try:
+            artifact_labels = utils.get_artifact_labels(variant=variant)
+
+            for iteration, label in enumerate(artifact_labels):
+                if iteration > 0:
+                    prior_label = artifact_labels[iteration - 1]
+                    prior_file_name = f"sbom.{prior_label}.json"
+                file_name = f"sbom.{label}.json"
+
+                if Path(file_name).is_file() and Path(file_name).stat().st_size > 0:
+                    LOG.info(f"Skipping {file_name} because it already exists...")
```

Comment:
This _should_ provide a decent speedup when testing locally, assuming short git commit hashes don't collide, and we don't re-release the exact same version (i.e. we release `2022.01.01` twice).  It is irrelevant in CI since all SBOM and Vuln scan artifacts are created from scratch.

Reply:
Makes sense.